wapm-toml/src/lib.rs

```diff
@@ -625,18 +658,37 @@ name = "mod"
 source = "target/wasm32-wasi/release/mod.wasm"
 interfaces = {"wasi" = "0.0.0-unstable"}
 
+[[module]]
+name = "mod-with-exports"
+source = "target/wasm32-wasi/release/mod-with-exports.wasm"
+bindings = { wit = "exports.wit", wit_bindgen = "0.0.0" }
```

Comment:
can we make it `wit-exports` instead of `wit`? It might make it more clear? Thoughts?

Reply:
I'm wary that this is one of those topics where you can keep going around and around because there's no perfect answer. 

For example, one could argue that the WebAssembly spec makes a very clear distinction between items that are imported and items that are exported, so `exports = { ... }` is more descriptive than `bindings = {...}`.

How about we use the current naming as a placeholder and next Monday or Tuesday we can figure out the final version on Slack? In the meantime, here are some other permutations I came up with.

```toml
exports = { wit = "exports.wit", wit-bindgen = "0.0.0" }
exports = { wit = "exports.wit", version = "0.0.0" }
exports = { wit = "exports.wit", wit-version = "0.0.0" }
bindings = { wit = "exports.wit", version = "0.0.0" }
bindings = { wit-exports = "exports.wit", version = "0.0.0" }
bindings = { exports = "exports.wit", wit-bindgen = "0.0.0" }
```